src/main/java/ToDoList.java

```diff
@@ -0,0 +1,106 @@
+import java.util.Arrays;
+
+public class ToDoList {
+    private Task[] taskList;
+    private String[] markedTask;
+    private int taskListTracker;
+    public String checker;
+    public String marker;
+    public String unmarked;
+    public String toDo;
+
+public ToDoList(){
+    taskList = new Task[100];
+    markedTask = new String[100];
+    taskListTracker = 0;
+    checker = "list";
+    marker = "mark";
+    unmarked = "unmark";
```

Comment:
Perhaps you could standardise the naming for these variables since they are similar, using noun for the variables.
```suggestion
    markCommand = "mark";
    unmarkCommand = "unmark";
```